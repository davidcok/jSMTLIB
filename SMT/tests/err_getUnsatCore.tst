; tests get-unsat-core without :produce-unsat-cores
(set-logic QF_UF)
(get-unsat-core)
(get-unsat-core xxx)
