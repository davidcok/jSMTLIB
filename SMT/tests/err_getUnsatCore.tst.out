success
(get-unsat-core)
^^^^^^^^^^^^^^^^
(error "The get-unsat-core command is only valid if :produce-unsat-cores has been enabled")
(get-unsat-core xxx)
                ^^^
(error "A get-unsat-core command takes no arguments")
